quantum/errors.js

```diff
@@ -127,9 +149,13 @@ module.exports = {
   QUBITS_FROM_DIFFERENT_CIRCUITS,
   SAME_QUBIT_RECEIVED_TWICE,
   BLOCH_SPHERE_WITH_MEASUREMENT,
+  GREATER_THAN_TWO,
+  INPUT_ODD_INTEGER,
+  INPUT_AN_INTEGER,
```

Comment:
```suggestion
```
Given that these are not used outside of `errors.js`, they don't necessarily need to be exported.